raiden/messages.py

```diff
@@ -1605,31 +1611,34 @@ def __init__(
             *,
             onchain_balance_proof: SignedBlindedBalanceProof,
             reward_amount: typing.TokenAmount,
-            non_closing_signature: typing.Signature = b'',
-            reward_proof_signature: typing.Signature = b'',
+            non_closing_signature: typing.Optional[typing.Signature] = None,
+            reward_proof_signature: typing.Optional[typing.Signature] = None,
```

Comment:
The constructor code was very cumbersome, it was replacing the default `b''` with `None` for no good reason.